README.md

```diff
@@ -54,9 +54,27 @@ Total: 10, Dialing: 1, Established: 9, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
 --- myhttpsamplehost.com:80 tcp test statistics ---
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
+Time spent in 10 established connections min/avg/max/stdDev = 17.929ms/19.814ms/29.811ms/3.353ms
 % echo $?
 0
 ```
+
+Partial unsuccesful execution
+```bash
+% ./tcpgoon --host myhttpsamplehost.com --port 8080 --connections 10 --sleep 999 -y
+Total: 10, Dialing: 0, Established: 0, Closed: 0, Error: 0, NotInitiated: 10
+Total: 10, Dialing: 9, Established: 1, Closed: 0, Error: 0, NotInitiated: 0
```

Comment:
I think the output does not match with the command executed. 9 dialing connections from one update to the other, with a 999ms sleep between connections, sounds strange :)

Reply:
xD, updated